benches/all.rs

```diff
@@ -27,7 +27,7 @@ macro_rules! bench_chars {
             });
 
             group.bench_function("to_byte_idx", |bench| {
-                let idx = $text.len() - 1;
+                let idx = chars::count($text) - 1;
```

Comment:
Ah! Thanks for this. Silly mistake. Looks like I made the same mistake in the utf16 and lines benchmarks as well, but I'll handle that separately so we don't clutter this PR.